src/types/api/CodiceIdRPT.ts

```diff
@@ -0,0 +1,37 @@
+/* tslint:disable:ordered-imports */

```

Comment:
if you generate this files at build time, you can put them in gitignore (for easier code reviews)